app/connector/FrontendAuthConnector.scala

```diff
@@ -18,11 +18,19 @@ package connector
 
 import config.WSHttp
 import model.CoAFEAuthority
-import play.api.libs.json.{Json, Reads}
+import play.api.libs.json.{Format, Json, Reads}
 import uk.gov.hmrc.play.config.ServicesConfig
 import uk.gov.hmrc.play.frontend.auth.connectors.AuthConnector
 import uk.gov.hmrc.play.http.HeaderCarrier
 
+case class EnrolmentIdentifier(key: String, value: String)
+case class GovernmentGatewayEnrolment(key: String, identifiers: Seq[EnrolmentIdentifier], state: String)
+
+object GovernmentGatewayEnrolment {
+  implicit val idFmt = Json.format[EnrolmentIdentifier]
```

Comment:
It is getting serialised. Removing the writes generates a compile error [here](https://github.com/hmrc/tax-account-router-frontend/blob/TSV-281/app/services/TwoStepVerification.scala#L70)